docs/admin/federation/kubefed.md

```diff
@@ -61,8 +60,8 @@ The output should contain an entry corresponding to your host cluster,
 similar to the following:
 
 ```
-CURRENT   NAME                                                               CLUSTER                                                            AUTHINFO                                                           NAMESPACE
-          gke_myproject_asia-east1-b_gce-asia-east1              gke_myproject_asia-east1-b_gce-asia-east1              gke_myproject_asia-east1-b_gce-asia-east1
+CURRENT   NAME                                          CLUSTER                                       AUTHINFO                                      NAMESPACE
+          gke_myproject_asia-east1-b_gce-asia-east1     gke_myproject_asia-east1-b_gce-asia-east1     gke_myproject_asia-east1-b_gce-asia-east1
```

Comment:
I still can't scroll this part. Maybe it's a website issue that needs to be fixed by @kubernetes/sig-docs?